packages/imperative/__tests__/__integration__/cmd/src/imperative/config.ts

```diff
@@ -281,10 +281,20 @@ export const config: IImperativeConfig = {
                         secure: true,
                         optionDefinition: {
                             name: "secret",
-                            description: "The secret info the keep in the profile.",
+                            description: "The secret info to keep in the profile.",
                             type: "string",
                             required: true,
                         }
+                    },
+                    undefined: {
+                        type: undefined,
+                        secure: true,
+                        optionDefinition: {
+                            name: "undefined",
+                            description: "The undefined secret info to keep in the profile.",
+                            type: undefined,
+                            required: true,
```

Comment:
(nit) I feel like we should avoid defining a secure property with a reserved keyword such as `undefined`, I know its in a test but it took me a little while to realize that `undefined` is just an example for the test 😅

Reply:
Resolved in 0a70678f4f3e6457bfa693dc0ef3fa7d75c2b835